pages/index.tsx

```diff
@@ -84,59 +73,59 @@ export const Wallet = (props: WalletProps) => {
         </ReactUIWalletModalProviderDynamic>
       </WalletProvider>
     </ConnectionProvider>
-  );
-};
+  )
+}
 
 type FaucetFormProps = {
-  showChooseNetwork: Boolean,
-  vm: ChainVm,
-  defaultFaucetUrl: string,
+  showChooseNetwork: Boolean
+  vm: ChainVm
+  defaultFaucetUrl: string
   account?: string | null
 }
 
+const shortenAddr = (address = '', length = 3) => {
+  if (!address) return ''
+
+  return `${address.slice(0, length)}...${address.slice(-length)}`
+}
+
 export const FaucetForm = (props: FaucetFormProps) => {
-  const {
-    showChooseNetwork,
-    vm,
-    defaultFaucetUrl,
-    account,
-  } = props
+  const { showChooseNetwork, vm, defaultFaucetUrl, account } = props
 
-  const [address, setAddress] = useState<string>("")
+  const [address, setAddress] = useState<string>('')
   const [amount, setAmount] = useState<string | null>(null)
   const [sending, setSending] = useState(false)
   const [faucetUrl, setFaucetUrl] = useState(defaultFaucetUrl)
   const [error, setError] = useState<string | null>(null)
   const [signature, setSignature] = useState<string | null>(null)
-  const { connection } = useConnection();
+  const [open, setOpen] = useState(false)
```

Comment:
Can we rename this for clarity:
```suggestion
  const [captchaOpen, setCaptchaOpen] = useState(false)
```